src/modules/0.0.22/aws_ecs_fargate/mappers/task_definition.ts

```diff
@@ -167,8 +167,8 @@ export class TaskDefinitionMapper extends MapperBase<TaskDefinition> {
     if (c.logConfiguration?.logDriver === 'awslogs') {
       const groupName = c.logConfiguration.options['awslogs-group'];
       const logGroup =
-        (await awsCloudwatchModule.logGroup.db.read(ctx, groupName)) ??
-        (await awsCloudwatchModule.logGroup.cloud.read(ctx, groupName));
+        (await awsCloudwatchModule.logGroup.db.read(ctx)).find((lg: any) => lg.logGroupName === groupName) ??
+        (await awsCloudwatchModule.logGroup.cloud.read(ctx)).find((lg: any) => lg.logGroupName === groupName);
```

Comment:
I'd leave a `TODO` here saying that it will need to be fixed once `task_definition` pass to be multi-region. If we have 2 log groups with the same name in different regions this could retrieve the wrong one if we do not filter by name and region.